src/components/profile/ProfileService.js

```diff
@@ -165,7 +165,7 @@ goog.require('ga_urlutils_service');
           $http(params).success(function(data, status) {
             // When all the geometry is outside switzerland
             if (data.length == 0) {
-              data = [{alts: {COMB: 0}, dist: 0}];
+              data = [{alts: {}, dist: 0}];
```

Comment:
@oterral the code was like that previsouly. We don't touch it, but I agree it seems not necessary to do it again.